<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginInstallDialog</class>
 <widget class="QWidget" name="PluginInstallDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugin Installation</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QStackedWidget" name="wizard">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="selectionPage">
      <layout class="QGridLayout">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;b&gt;Enter the plugin archives to install&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="3">
        <widget class="QListWidget" name="archivesList">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="addArchivesButton">
         <property name="toolTip">
          <string>Add plugin ZIP-archives via a file selection dialog</string>
         </property>
         <property name="text">
          <string>Add ...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="removeArchivesButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Remove the selected entries from the list of plugin archives to be installed</string>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>101</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="destinationPage">
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;b&gt;Select the destination plugin directory&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="destinationCombo">
         <property name="toolTip">
          <string>Select the destination plugin area</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="summaryPage">
      <layout class="QVBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&lt;b&gt;Installation Summary&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="summaryEdit">
         <property name="toolTip">
          <string>This shows the summary of the installation data</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress">
         <property name="value">
          <number>0</number>
         </property>
         <property name="format">
          <string>%v/%m Files</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>addArchivesButton</tabstop>
  <tabstop>archivesList</tabstop>
  <tabstop>removeArchivesButton</tabstop>
  <tabstop>destinationCombo</tabstop>
  <tabstop>summaryEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
